projects/uitpas/reference/uitpas.json

```diff
@@ -6403,23 +6410,39 @@
           },
           "socialTariff": {
             "type": "object",
-            "description": "If the passholder has right to a social tariff, this object contains details like the end date.",
+            "description": "If the passholder has (or had) right to a social tariff, this object contains details like the  end date. Check the `status` of the `socialTariff`: the passholder is only currently entitled to a social tariff is `status` is `ACTIVE`. ",
             "properties": {
+              "status": {
+                "type": "string",
+                "enum": [
+                  "ACTIVE",
+                  "EXPIRED",
+                  "SUSPENDED"
+                ],
+                "description": "Status of the social tariff:\n- `ACTIVE`: the passholder is entitled to social tariff\n- `EXPIRED`: the passholder is NOT entitled to social tariff anymore\n- `SUSPENDED`: the passholder is temporarily NOT entitled to social tariff. In this case a extra property `suspendedUntilDate` will also be available."
+              },
               "endDate": {
                 "type": "string",
                 "format": "date-time",
-                "description": "Exact moment that the passholder's right to a social tariff expires."
+                "description": "Exact moment the passholder's right to a social tariff expires."
               },
               "inGracePeriod": {
                 "type": "boolean",
                 "description": "When the end date of the right to a social tariff has passed, the passholder may still be in a grace period that they can buy tickets at a social tariff until their right to a social tariff has been renewed."
               },
+              "suspendedUntilDate": {
+                "type": "string",
+                "format": "date-time",
+                "description": "Date until this social tarif is suspended (only available when status is `SUSPENDED`)"
```

Comment:
Formulering lijkt me een beetje dubbelzinnig. Mogelijk alternatief:

Date the social tariff suspension ends and the user can resume using its card.

(detail)